.travis.yml

```diff
@@ -1,14 +1,17 @@
 language: rust
-
 rust:
     - stable
     - beta
     - nightly
 
+cache: cargo
+before_script: (cargo install rustfmt || true)
 script:
-    - cargo doc --no-deps -p amethyst -p amethyst_ecs -p amethyst_config -p amethyst_context
-    - cargo build -v
-    - cargo test -p amethyst -p amethyst_ecs -p amethyst_config -p amethyst_context -v
+    - |
+      cargo fmt -- --write-mode=diff &&
```

Comment:
I think you need to add `PATH="$PATH:/home/travis/.cargo/bin"` before `cargo fmt`